src/components/card/CardSidebar.vue

```diff
@@ -170,11 +175,46 @@
 			icon="icon-activity">
 			<CardSidebarTabActivity :card="currentCard" />
 		</AppSidebarTab>
+		<Modal v-if="modalShow" :title="t('deck', 'Choose attachment')" @close="modalShow=false">
```

Comment:
This could actually also be a separate componen as the CardSidebar is already quite huge.